cm-vault/cm-config/main.tf

```diff
@@ -43,6 +43,13 @@ resource "kubernetes_manifest" "vault-issuer" {
       }
     }
   }
+
+  wait {
```

Comment:
Suggestion: should we add a timeout? I see that it takes quite long for the operation to time out.
I think `60s` should be sufficient for all operations.
Same with the `Certificate` resource below.

Reply:
Thanks for remind. I have added it.